
Wed, 09 Jun 2010 07:37:06 GMT

COMMAND LINE:
/usr/local/src/pypy-1.2/bin/pypy spectralnorm.pypy-5.pypy 500

PROGRAM FAILED 


PROGRAM OUTPUT:

Traceback (most recent call last):
  File "?", line 33, in run_toplevel
  File "spectralnorm.pypy-5.pypy", line 9, in <module>
    from multiprocessing import Pool
ImportError: No module named multiprocessing
